src/js/control-bar/playback-rate-menu/playback-rate-menu-button.js

```diff
@@ -78,22 +78,18 @@ class PlaybackRateMenuButton extends MenuButton {
   }
 
   /**
-   * Create the playback rate menu
+   * Create the list of menu items. Specific to each subclass.
    *
-   * @return {Menu}
-   *         Menu object populated with {@link PlaybackRateMenuItem}s
    */
-  createMenu() {
-    const menu = new Menu(this.player());
+  createItems() {
     const rates = this.playbackRates();
+    const items = [];
 
-    if (rates) {
-      for (let i = rates.length - 1; i >= 0; i--) {
-        menu.addChild(new PlaybackRateMenuItem(this.player(), {rate: rates[i] + 'x'}));
-      }
+    for (let i = rates.length - 1; i >= 0; i--) {
+      items.push(new PlaybackRateMenuItem(this.player(), {rate: rates[i] + 'x'}));
```

Comment:
This should probably be localised with `rates[i].toLocaleString()`, because the decimal separator in many locales is a comma.  That does complicate things, `PlaybackRateMenuItem` will use `parseFloat()` on this string but that only works with a decimal point regardless of locale. Is it better to pass the float here and have `PlaybackRateMenuItem` set its label to something localised instead? Perhaps the 'x' also should be localisable, e.g. 1.5倍.

Reply:
Making this more accessible does make sense. I doubt it will affect anyone but changing `rate` from a string to a number would be a technically breaking change, but might be an acceptable soft-breakage. It definitely makes sense for the MenuItem to handle how it wants to do things itself.
Probably should have an issue to discuss this further, I don't think it should be part of this PR.

Reply:
It could be passed through as it currently is to not be breaking, and the Menu item could still parse and localise it. But yes makes sense to be a separate change.